File: product-webapp/webapp/src/project.ts
import { Status } from './status';

export class Project {
  constructor(
    public projectName: string,
    public organizationName: string,
    public startDate: string,
    public endDate: string,
    public status: Status,
    public managerEmailId: string,
    public members: string[],
    public image: string
  ) {}
}

File: time-tracker-service/src/main/java/com/stackroute/timetrackerservice/dto/project/Project.java
package com.stackroute.timetrackerservice.dto.project;


import lombok.Data;
import lombok.NoArgsConstructor;

import java.util.List;

@Data
@NoArgsConstructor
public class Project {

    public static final String SEQUENCE_NAME = "projects_sequence";
    private long projectId;
    private String projectName;
    private int projectCode;
    private byte[] organizationBrandLogo;
    private long createdOn;
    private long startDate;
    private long endDate;
    private String managerEmailId;
    private List<Member> membersList;
    private ProjectStatus projectStatus;
    private int costCode;
    private int customerId;
}
